test/e2e/pages/vl-accordion.page.js

```diff
@@ -35,6 +31,11 @@ class VlAccordionPage extends Page {
         return this._openAccordion('#accordion-1');
     }
 
+    async getStandaardAccordionTextContent() {
+        return this.driver.executeScript('return arguments[0].textContent',
+            (await (await this.getStandaardAccordion()).contentElements())[0]);
```

Comment:
heb je hier javascript voor nodig? bestaat er geen getText() op webdriverelement?

Reply:
Had ik geprobeerd, maar leek niet te werken

Reply:
Selenium Webdriver heeft een getText functie: `Get the visible (i.e. not hidden by CSS) innerText of this element, including sub-elements, without any leading or trailing whitespace.`.